plugin/path_role_set_secrets.go

```diff
@@ -0,0 +1,144 @@
+package gcpsecrets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+var fieldSchemaRoleSetServiceAccountKey = map[string]*framework.FieldSchema{
```

Comment:
nit: Perhaps we can return the values set to `fieldSchemaRoleSetServiceAccountKey` and `fieldSchemaRoleSetAccessToken` from functions instead of introducing global vars? That would reduce the potential for modification at run time.

Reply:
Done in17c1921